examples/usecases/transformers-next-item-prediction-with-pretrained-embeddings.ipynb

```diff
@@ -0,0 +1,731 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    cd /workspace &amp;&amp; pip install . 
</pre><p>why do we need to do pip install here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/1102/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
I am modifying `merlin-models` to include the schema for the new datasets (`sigir-browsing` and `sigir-sku`) . In order to make them available to me in the notebook, I need to install the modified library (which inside docker lives in `/workspace`). I am just doing this to help with working on the PR but you are right, this should not exist in the final version.